pkg/fission-cli/flag/flag.go

```diff
@@ -116,7 +116,7 @@ var (
 	FnSubPath               = Flag{Type: String, Name: flagkey.FnSubPath, Usage: "Sub Path to check if function internally supports routing"}
 
 	HtName              = Flag{Type: String, Name: flagkey.HtName, Usage: "HTTP trigger name"}
-	HtMethod            = Flag{Type: String, Name: flagkey.HtMethod, Usage: "HTTP Method: GET|POST|PUT|DELETE|HEAD", DefaultValue: http.MethodGet}
+	HtMethod            = Flag{Type: StringSlice, Name: flagkey.HtMethod, Usage: "HTTP Methods: GET,POST,PUT,DELETE,HEAD. To mention single method: --method GET and for multiple methods --method GET --method POST.", DefaultValue: []string{http.MethodGet}}
```

Comment:
Can we have a `Methods` flag and deprecate the `Method` flag here? It would be nicer from user perspective to just do : `--methods=GET,POST`

Reply:
This would be a wider changes if we change name of flag. Since this is targeted towards limited users as of now, I think we should be good.
Even if we get specific request of `GET,POST` kind of syntax we can support it within existing flag. Where both `GET,POST` and `--method GET --method POST` formats can be supported. So I dont see need of overriding existing flags. It can accept existing as well as two formats discussed.